packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -378,38 +382,53 @@ fn schema_to_stream_bytes(schema: &Schema) -> Vec<u8> {
     stream_bytes
 }
 
-fn array_to_errors(array: mv8::Value<'_>) -> Vec<RunnerError> {
-    // TODO: Extract optional line numbers
+fn array_to_user_errors(array: mv8::Value<'_>) -> Vec<UserError> {
```

Comment:
Don't we want to extract the line numbers here as well?

Reply:
There isn't a line number field on the UserError message.
```
// fields:
//    `msg` : the error message
table UserError {
  msg:string (required);
}
```